src/componentsTest/MyEvents.js

```diff
@@ -0,0 +1,612 @@
+            setEventLocationErrorText('Please enter the location of your event');
+            return false;
+        } else if (eventParticipants === '') {
+            setEventParticipantsError(true);
+            setEventParticipantsErrorText('Please enter a valid number of maximum participants.');
+            return false;
+        } else if (eventParticipants < eventCurrentParticipants) {
+            setEventCurrentParticipantsError(true);
+            setTimeout(() => {
+                setEventCurrentParticipantsError(false);
+            }, 10000);
+        } else {
+            CallApiEditEvent();
+            setEventCurrentParticipantsError(false);
+            setShowEditAlertMessage(true);
+            setTimeout(() => {
+                window.location.reload();
+            }, 3000);
+        }
+    }
+
+    const handleGetParticipants = () => {
+        CallApiGetParticipants()
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+                //console.log(parsed[0]);
+                //setParticipantsList(parsed);
+                setSelectedEventParticipants(parsed); 
+            });
+    };
+
+    const CallApiGetParticipants = async () => {
+
+        const url = `${REACT_APP_API_ENDPOINT}/getParticipants`;
+        //console.log(url);
+        //console.log("selectedEvent = " + selectedEvent)
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+              body: JSON.stringify({
+               eventID: currentEventID
+             })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    };
+
+    /*
+    useEffect(() => {
+        loadGetEventsByUser();
+    }, [userID]);
+*/
+    const classes = useStyles();
+
+    useEffect(() => {
+        loadGetMyEvents();
+      }, [loadGetMyEvents]);
+
+    return (
+        <div id="body">
+
+            <Typography
+                variant="h4"
+                gutterBottom
+                component="div">
+                My Events
+            </Typography>
+
+            {events.map((event) => (
+                <Card style={{ width: '500px' }} key={event.id}>
+                    <CardContent>
+                        <Typography variant="h5" component="div">
+                            {event.name}<br />
+                        </Typography>
+                        <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                            Date: {new Date(event.date).toLocaleDateString()}<br />
+                        </Typography>
+                        <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                            Participants: {event.participants} / {event.totalParticipants}<br />
+                        </Typography>
+                        <Typography sx={{ mb: 1.5 }} color="text.secondary">
```

Comment:
remove the three `color="text.secondary"` style parameters, they're throwing warnings in the tests